src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -624,38 +625,38 @@ public partial class RetryConditionHeaderValue : System.ICloneable
         public RetryConditionHeaderValue(System.TimeSpan delta) { }
         public System.DateTimeOffset? Date { get { throw null; } }
         public System.TimeSpan? Delta { get { throw null; } }
-        public override bool Equals(object obj) { throw null; }
+        public override bool Equals(object? obj) { throw null; }
         public override int GetHashCode() { throw null; }
-        public static System.Net.Http.Headers.RetryConditionHeaderValue Parse(string input) { throw null; }
+        public static System.Net.Http.Headers.RetryConditionHeaderValue Parse(string? input) { throw null; }
         object System.ICloneable.Clone() { throw null; }
         public override string ToString() { throw null; }
-        public static bool TryParse(string input, out System.Net.Http.Headers.RetryConditionHeaderValue parsedValue) { throw null; }
+        public static bool TryParse(string? input, out System.Net.Http.Headers.RetryConditionHeaderValue? parsedValue) { throw null; }
```

Comment:
Does this TryParse method not guarantee the parsedValue is not null if the method returns true?  Same question for some of the others below missing the attribute.

Reply:
yes it is